examples/filesystem/README.md

```diff
@@ -1,31 +1,71 @@
 # File system usage example
 
-This is a basic example how to use a file system with RIOT in your embedded application.
+## Description
 
-This example shows:
- - how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs examples
- - how to open/read/write/close a file with and without newlib
+This basic example shows how to use a file system with RIOT in your embedded
+application.
+
+In particular, this example shows:
+
+- how to mount/format/unmount a file system, either with spiffs, littlefs or
+  constfs
+- how to open/read/write/close a file with and without newlib
 
 In RIOT, most filesystems use `mtd` as flash interface. So to use this example
-you must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
-This example use `littlefs` as default filesystem on the whole `mtd` device.
-A `constfs` filesystem is also demonstrated with two files.
+one must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
 
-All the RIOT filesystems are used through the `vfs` interface, and on most platform
-they can be accessed transparently with `open/close/read/write/...` functions.
+This example uses `littlefs` as default file system on the whole `mtd` device.
+A `constfs` file system is also demonstrated with two files.
+
+All the RIOT file systems are used through the `vfs` interface, and on most
+platforms they can be accessed transparently with `open/close/read/write/...`
```

Comment:
Did you address my comment here?

Reply:
Well, the actual sentence is clear enough. Maybe @vincent-d can give his opinion ?

Reply:
For the record @kYc0o is referring to [this comment](https://github.com/RIOT-OS/RIOT/pull/8944#discussion_r181341175)